src/Entity/PhoneNumberSettings.php

```diff
@@ -56,7 +56,7 @@ class PhoneNumberSettings extends ConfigEntityBase implements PhoneNumberSetting
    *
    * @var string
    */
-  protected $entity_type;
+  protected $entityType;
```

Comment:
Commented at https://www.drupal.org/node/2643692#comment-10849714. We can revert those hunks.